ddtrace/encoding.py

```diff
@@ -1,15 +1,15 @@
 from .internal.encoding import Encoder
 from .internal.encoding import JSONEncoder
 from .internal.encoding import JSONEncoderV2
-from .internal.encoding import MsgpackEncoder
+from .internal.encoding import MsgpackEncoderV03
 from .utils.deprecation import deprecation
 
 
 __all__ = (
     "Encoder",
     "JSONEncoder",
     "JSONEncoderV2",
-    "MsgpackEncoder",
+    "MsgpackEncoderV03",
```

Comment:
Breaking API change

Reply:
Yep, this is still a draft and not totally clear to me at the moment what will happen with naming. I'll sort this out once this  is clear so I don't end up changing it over and over 🙂